excercises/4-pop/contact.spec.ts

```diff
@@ -8,8 +8,9 @@ describe('Protractor Workshop app', function() {
 
 	it('should display text "Your message has been sent." when user sends a message', async function(){
 		const success = "Your message has been sent."
-		await contactPage.fillUpFormAndSubmitt("name noname","name.noname@gmail.com","content");		
-		expect(await contactPage.getSuccessInformation()).toEqual(success);
+		await contactPage.fillUpFormAndSubmitt("name noname","name.noname@gmail.com","content");
+		let getSuccess = await element(by.css(".alert-success"))	
+		expect(getSuccess.isPresent()).toBe(true);	
```

Comment:
I think the previous way was actually better. That way you not only check if the success alert is displayed but also if it has expected text.